src/utils/axiosSetup.test.ts

```diff
@@ -54,14 +54,18 @@ describe('axiosSetup', () => {
     } as unknown as AxiosError;
 
     const mockErrorRefreshFailed: AxiosError = {
-      config: {},
+      config: {
+        headers: {} as AxiosRequestHeaders,
+      },
       request: {},
       response: {
         data: { error_description: 'The provided refresh token is invalid.', error: 'invalid_grant' },
         status: 497,
         statusText: 'unknown status',
         headers: {},
-        config: {},
+        config: {
+          headers: {} as AxiosRequestHeaders,
```

Comment:
Sure that response headers are type of AxiosRequestHeaders?

Reply:
I think so, but I'm open to any better/further ideas